base_layer/core/src/base_node/rpc/service.rs

```diff
@@ -597,4 +599,30 @@ impl<B: BlockchainBackend + 'static> BaseNodeWalletService for BaseNodeWalletRpc
 
         Ok(Streaming::new(rx))
     }
+
+    async fn get_mempool_fee_per_gram_stats(
+        &self,
+        request: Request<GetMempoolFeePerGramStatsRequest>,
+    ) -> Result<Response<GetMempoolFeePerGramStatsResponse>, RpcStatus> {
+        let req = request.into_message();
+        let count =
+            usize::try_from(req.count).map_err(|_| RpcStatus::bad_request("count must be less than or equal to 20"))?;
```

Comment:
Does usize only go to 20? :stuck_out_tongue_closed_eyes:

Reply:
lol the fact remains, it must be less than 20 :)